fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -981,6 +981,10 @@ alter_table_clause ::=
     {:
         RESULT = new EnableFeatureClause(featureName);
     :}
+    | KW_ENABLE KW_FEATURE STRING_LITERAL:featureName KW_WITH opt_properties:properties
+    {:
+        RESULT = new EnableFeatureClause(featureName, properties);
```

Comment:
you may add an opt_enable_feture_properties and merge them  to `KW_ENABLE KW_FEATURE STRING_LITERAL:featureName opt_enable_feture_properties`

Reply:
Done